### DISCUSSION TITLE
sp-samples-1k.log cannot be downloaded normally

### DISCUSSION THREAD

**Discussion by messixukejia** [fluent/fluent-bit#6857] [Q&A] [answered]
The example [file sp-samples-1k.log](https://fluentbit.io/samples/sp-samples-1k.log) in [Stream Processing](https://docs.fluentbit.io/manual/stream-processing/getting-started/hands-on) can not be downloaded normally. Where can I download it?

**Comment by patrick-stephens** [comment, marked as answer]
Likely down to migration of the legacy server  so I'll see if I can find it in the backup and update the docs.

https://github.com/fluent/fluent-bit/discussions/6688

**Reply by patrick-stephens** [reply]
This is the file, I'll see if I can update the docs to host it. [sp-samples-1k.log](https://github.com/fluent/fluent-bit/files/10771424/sp-samples-1k.log)

**Reply by patrick-stephens** [reply]
@messixukejia I've updated the docs to host it directly now.
